src/test/benchmarks/contracts/escrow.tact

```diff
@@ -146,7 +152,7 @@ contract Escrow with Deployable {
     }
 
     // send funds back to buyer
-    receive("cancel"){
+    receive(_: Cancel){
         require(self.isFunded == true, "not funded");
```

Comment:
let's use `throwUnless` instead of `require`, so the exit codes are also equivalent to the ones used in the FunC version